frontend/package.json

```diff
@@ -21,7 +21,9 @@
 		"playwright:codegen:local": "playwright codegen http://localhost:3301",
 		"playwright:codegen:local:auth": "yarn playwright:codegen:local --load-storage=tests/auth.json",
 		"husky:configure": "cd .. && husky install frontend/.husky && cd frontend && chmod ug+x .husky/*",
-		"commitlint": "commitlint --edit $1"
+		"commitlint": "commitlint --edit $1",
+		"test": "jest --coverage",
+		"test:changedsince": "jest --changedSince=develop --coverage --silent"
```

Comment:
We should also define the limits at which the check should fail, else it won't be paid attention to. We can keep a lower limit initially and keep increasing it over time

Reply:
added - 

`coverageThreshold: {
		global: {
			statements: 65,
			branches: 50,
			functions: 50,
			lines: 65,
		},
	},`